api/v1/openapi.yaml

```diff
@@ -176,13 +176,7 @@ paths:
       tags:
       - endpoint
       parameters:
-      - name: labels
-        description: |
-          List of labels
-        in: body
-        required: false
-        schema:
-          "$ref": "#/definitions/Labels"
+      - "$ref": "#/parameters/labels"
```

Comment:
The explanation in the commit is not sufficient - what makes labels "not really required" anymore?

Reply:
`cilium identity get` which makes calls `v1/identity` does not send any labels, the labels were always optional. swagger 0.12.0 was never refusing a request when labels were not specified, but that is not the case with v0.18.0 it is the same of #7029